src/conf_mode/containers.py

```diff
@@ -215,20 +215,32 @@ def apply(container):
                 # Currently the best way to run a command and immediately print stdout
                 print(os.system(f'podman pull {image}'))
 
+                # Bind volume
+                volume = ''
+                if 'bind_volume' in container['name'][name]:
```

Comment:
There is already `container_config` config in place which has the same content as `container['name'][name]`. Please adjust the syntax.

Reply:
done